### ISSUE TITLE
Error in new fresh install: column 'country' can't have a default value

### ISSUE THREAD

**Issue by cdiaz23** [Jefferson49/joomla4_plugin_sexy_polling#49]
Hi!

Installing joomla_plugin_sexypolling_reloaded_v4.1.7.zip in a Joomla! 3.10.12, we get this error:

```
Advertencia
BLOB/TEXT column 'country' can't have a default value
×
Error
Error instalando componente
```
<a href="https://imgbb.com/"><img src="https://i.ibb.co/6m8YbbW/error.png" alt="error" border="0"></a>
<a href="https://ibb.co/wCnM2Sf"><img src="https://i.ibb.co/xGnJr1N/eror2.png" alt="eror2" border="0"></a>

**Comment by Jefferson49** [comment]
Thank you for reporting the issue!

In issue #46, a similar topic was reported by a user with a MySQL data base version before 5.7.24 (or 8.0.13). Can you check and post your database version? 

If possible with your server/provider, you might want to update your MySQL database to a newer version (5.7.24/8.0.13) or later.

If you do not want to update the database, you can install the former v4.1.6 release of the plugin. From v4.1.6 to v4.1.7, there was a change in the code regarding database default values, which might not be compatible to former database versions.

**Comment by cdiaz23** [comment]
Many thanks @Jefferson49 !

Mysql database version is 5.5.45 (too old). We'll try 4.1.6 version by now.

Thanks!

**Comment by Jefferson49** [comment]
Was the installation with v4.1.6 succesful?

**Comment by cdiaz23** [comment]
Sorry for delay @Jefferson49 

Yes! 4.1.6 version is ok for us! Thanks

**Comment by Jefferson49** [comment]
Just recently, I published a new release [v5.0.0](https://github.com/Jefferson49/Joomla_plugin_sexypolling_reloaded/releases/tag/v5.0.0), which is avoiding installation errors of the tpye "**column 'country' can't have a default value**". Maybe, you can check if the new version also solves the issues you described.

**Comment by Jefferson49** [comment]
Hopefully fixed with v5.0.0
